src/App.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Profile from './components/Profile';
+import Rockets from './components/Rockets';
+import Dragons from './components/Dragons';
+import Missions from './components/Missions';
+
+const App = () => (
+  <Router>
+    <Navbar />
+    <Routes>
+      <Route exact path="/" element={<Profile />} />
+      <Route path="/rockets" element={<Rockets />} />
+      <Route path="/missions" element={<Missions />} />
+      <Route path="/dragons" element={<Dragons />} />
+    </Routes>
```

Comment:
| Testing Requirement |
|-|
| ![image](https://github.com/danielochuba/space-travelers-hub/assets/93039359/825ff835-5586-49c9-8c03-b12dfa86e127) | 
| ![image](https://github.com/danielochuba/space-travelers-hub/assets/93039359/d4e470c5-757b-41af-8b67-6dee54a14059) |


- [ ] It would be nice if you make tests for your app to verify that your app is working as expected 👍